### DISCUSSION TITLE
Granular Codespace Spend

### DISCUSSION THREAD

**Discussion by MainViking** [community/community#38608] [Codespaces]
Hello, I wanted to know if there are any plans to provide organizations with more granular control for Codespace spend. Currently limits can only be set for all Codespaces, instead of at the individual Codespace level. If not available where can I submit feedback for the ability to set a specific dollar amount or a maximum number of hours for individual Codespaces? Thanks!

**Comment by otter-computer** [comment]
Hi @MainViking, welcome to the GitHub Community! You're in the right place to post feedback about Codespaces. If you have any more specific details about this feature request then feel free to leave them in this thread. The Codespaces team will see it here!

**Reply by MainViking** [reply]
Thanks @otter-computer. Essentially, we are looking to limit the monthly spend for individual Codespaces. i.e.
Codespace1: $30/mo. spend limit
Codespace2: $20/mo. spend limit

Currently, we are only able to set the monthly spend limit for all Codespaces, not individual. Having the flexibility to limit spend per Codespace would enable specific teams to set budget constraints on a per repository basis.
